Tests/DatadogTests/Datadog/RUM/RUMMonitor/Scopes/RUMViewScopeTests.swift

```diff
@@ -548,6 +548,74 @@ class RUMViewScopeTests: XCTestCase {
         XCTAssertEqual(event.model.view.action.count, 2, "View should record 2 actions: non-custom + instant custom")
     }
 
+    func testGivenViewWithPendingAction_whenCustomActionIsAdded_itSendsItInstantly() throws {
```

Comment:
These two tests ensure we track `.custom` actions properly, no matter if added while any action scope exists or not. In both cases, `.custom` action is sent instantly w/o waiting for their side effects.